index.rst

```diff
@@ -4,9 +4,93 @@ Night Planning Tool
+   This is a tools that allows our team to collect a group of Jira tickets describing a set of tests/observations, allows us to organize these into a coherent list that can later be loaded and executed at the summit.
 
-Add content here
-================
+Introduction
+============
+
+Test planning and execution for commissioning the Vera Rubin observatory has revolved around using Jira and Confluence to generate a comprehensive plan that can be followed by the night crew.
+This process involves a combination of Jira projects:
+
+- SITCOM are used to capture high level tests/verification activities.
+  They are usually connected to one or more requirement or some other commissioning activity.
+  For example, SITCOM-1099 was created to capture the tests required to validate the operation of the system with the TMA dynamic settings going from 5% all the way to 100% of its requirements.
+
+- DM tickets are used to capture work needed either by a regular contruction deliverable or a commissioning activity.
+  DM tickets can be linked to SITCOM tickets when there is work required to accomplish a particular test, but they can also be standalone ticket that are just broadly required.
+  For instance, in the example above of SITCOM-1099, we have DM-39533 linked as a related ticket.
+  It is not strickly necessary to have this relantionship as DM-39533 represents a broader system functionality, however the issues are linked to demonstrate the relantionship.
+  Sometimes a SITCOM ticket might trigger the development of systewide functionality and the relantionship is used to indicate it.
+
+- BLOCK tickets are used to design, plan, execute and track activities at the summit.
+  Usually a BLOCK ticket will be linked to a SITCOM ticket.
+
+Up to now, planning activities at the summit have revolved around collecting the BLOCK tickets that will be executed at night and sorting them out in a table with some information about priority and order of execution.
+Later, the summit crew adds information to these tables about the execution and are also required to do more detailed logging in LOVE/OLE. 
+Gathering information later about task execution and performing traceability under these conditions is taxing.
+Users have to peruse the night logs for detailed information and also check the confluence page with the night plan.
+
+In order to improve overal user experience on both sides of the isle; creating and executing the plan, and also to improve metadata extraction for traceability, we propose to introduce a set of new procedures and tools for planning and execution of the summit activities.
+ 
+Planning with Zephyr
+====================
+
+We will adopt Jira's Zephyr application to create test cases, test cycles and test plans to organize how tests will be executed and planned.
+
+- BLOCK tickets will be used to describe what the block does, link it to a particular SITCOM ticket and will also aggregate the block configuration.
```

Comment:
If we adopt Zephyr, I recommend that we limit BLOCK tickets for those who will have JSON files or scheduler configurations. I don't see why we need to create a BLOCK ticket **and** a test case if that does not favor automatization/scripting.

Reply:
I am not sure about this. For example, see the doc page that Erik is working on describing the BLOCK project workflow. I think there is some value to having a BLOCK ticket. However, this might be a good discussion to have with the rest of the team.

Reply:
Can we have a couple of examples here? I believe, the way this is now, it is too abstract for a discussion.

Reply:
I think Erik's document would be a better place for this information. I am going to link Erik's document here and will add an item in the FAQ to talk about this.